examples/fabric-admin/BUILD.gn

```diff
@@ -80,6 +80,8 @@ static_library("fabric-admin-utils") {
     "commands/pairing/OpenCommissioningWindowCommand.h",
     "commands/pairing/PairingCommand.cpp",
     "commands/pairing/ToTLVCert.cpp",
+    "device_manager/BridgeAdminDeviceMapper.cpp",
```

Comment:
I am open to alternative naming suggestion. I always find naming hard

Reply:
ScopedNodeIdHandleMapper?  Reflects the role of mapping between ScopedNodeId and handles. But I am not clear why can't we just use ScopedNodeId directly.